executor/src/main/scala/org/apache/amaterasu/executor/execution/actions/runners/spark/PySpark/PySparkRunner.scala

```diff
@@ -28,7 +29,7 @@ import org.apache.amaterasu.sdk.AmaterasuRunner
 import org.apache.spark.SparkEnv
 import org.apache.spark.sql.SparkSession
 
-import scala.sys.process.Process
+import scala.sys.process._
```

Comment:
As a convention, it is recommended to avoid import entire libraries (as in python it is kind of a code-smell to do 'from foo import *'. Please specify specific classes/objects etc to import.